docs/nodejs/nodejs-debugging.md

```diff
@@ -315,7 +315,7 @@ If you want to attach to a Node.js process that hasn't been started in debug mod
 }
 ```
 
-Since it is a bit laborious to repeatedly find the process ID and enter it in the launch configuration, Node debug supports a command variable `PickProcess` that will open the process picker (from above).
+To avoid to enter a new process ID in the launch configuration, Node debug supports a command variable `PickProcess` that will open the process picker (from above).
```

Comment:
Suggest: "To avoid repeatedly entering a new process ID in the launch configuration,..."

Reply:
Updated in follow up PR https://github.com/microsoft/vscode-docs/pull/6339